src/pages/blog/article/index.js

```diff
@@ -0,0 +1,172 @@
+    border-radius: 8px;
+    position: relative;
+    overflow: hidden;
+
+    @media ${device.tabletL} {
+        max-width: 360px;
+        flex-direction: column;
+        padding: 24px 16px 45px;
+    }
+`
+const BackgroundImageWrapper = styled.div`
+    position: absolute;
+    bottom: -2px;
+    left: 0;
+    z-index: 1;
+`
+const BackgroundImage = styled.img`
+    width: 100%;
+`
+const ContentWrapper = styled(Flex)`
+    z-index: 3;
+    @media ${device.tabletL} {
+        justify-content: center;
+        flex-direction: column;
+        max-width: 360px;
+    }
+`
+const ImageWrapper = styled(Flex)`
+    width: 180px;
+    overflow: hidden;
+    @media ${device.tabletL} {
+        width: 218px;
+        padding-top: 0;
+        margin-bottom: 16px;
+    }
+`
+const ImgDiv = styled.img`
+    width: 100%;
+`
+const TextWrapper = styled(Flex)`
+    width: auto;
+    @media ${device.tabletL} {
+        margin-bottom: 16px;
+    }
+`
+const CTAButton = styled(LinkButton)`
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    line-height: 20px;
+    background-color: ${(props) => props.background_color && props.background_color};
+    color: ${(props) => props.color && props.color};
+    border-color: ${(props) => props.border_color && props.border_color};
+`
+const DesktopImageWrapper = styled(Container)`
+    width: 100%;
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
+const MobileImageWrapper = styled(Container)`
+    display: none;
+    @media ${device.tabletL} {
+        width: 100%;
+        display: flex;
+    }
+`
+
+// all commented value is by default from the code base,only if the value is not provided
+const item_data = {
+    banner_image: { mcBookPro },
+    banner_image_alt: 'Article CTA Banner Image',
+    background: {
+        // color: '#F8FAFB',
+        background_img: { bgImages },
+        background_img_mobile: { bgImagesMobile },
+    },
+    main_title: 'Join over 1 million traders worldwide',
+    small_desc: 'Sign up for an account now.',
+    button: {
+        // background_color: '#FF444F',
+        // border_color: '#FF444F',
+        text: 'Try free demo',
+        redirect_link: '/signup/',
+        // text_color: '#FFFFFF',
+    },
+}
+
+const BottomCta = () => {
+    return (
+        <Layout>
+            <ParentWrapper>
+                <MainWrapper
+                    p="0 32px 0 24px"
+                    max_width="792px"
+                    background_color={item_data.background.color}
+                >
+                    <ContentWrapper ai="center" jc="space-between">
+                        <ImageWrapper pt="12px">
+                            <ImgDiv src={mcBookPro} alt={item_data.banner_image_alt} />
+                        </ImageWrapper>
+                        <TextWrapper direction="column">
+                            <Header
+                                as="p"
```

Comment:
Let's use h2/h3 tag for this header title

Reply:
changed in latest commit